src/khoj/processor/content/markdown/markdown_to_entries.py

```diff
@@ -31,15 +30,14 @@ def process(
         else:
             deletion_file_names = None
 
+        max_tokens = 256
```

Comment:
Yeah, ideally should be coming from model, more so now that there are search models with larger context size (previously most supported 512 max tokens because that's default for BERT). But the models don't currently expose this. We should add this to the `SearchModelConfig` to make it configurable by admins